### ISSUE TITLE
Highlight arrow function parameters

### ISSUE THREAD

**Issue by dbaeumer** [Microsoft/TypeScript-TmLanguage#22]
_From @Bigous on December 4, 2015 12:48_

When you are writing in TypeScript, arrow function does not get the parameter names colored. Using anonymous functions you get it colored.

_Copied from original issue: Microsoft/vscode#1013_

**Comment by dbaeumer** [comment]
_From @alexandrudima on December 4, 2015 14:27_

@Bigous can you provide the code snippet please.
@dbaeumer this might be an upstream issue in the TypeScript TM grammar

**Comment by dbaeumer** [comment]
_From @Bigous on December 4, 2015 16:47_

Sure...
This is not colored
```typescript
app.get('/test/', (req, res) => {
});
```
This is colored correctly
```typescript
app.get('/test/', function (req, res) {
});
```
Just like here in github :D

[]s

**Comment by sheetalkamat** [comment]
Fixed by #257
